ModelViewer/ModelViewer/ViewerGraphicsWindow.h

```diff
@@ -26,6 +26,21 @@ class ViewerGraphicsWindow : public OpenGLWindow
     bool addPrimitive(QString filepath);
     bool IsModelValid();
 
+    bool editCurrentShaders();
+    bool reloadCurrentShaders();
+    bool openShaderFile(QString filepath = QString());
+
+    // Mouse variables
+    QMatrix4x4 sceneMatrix;
```

Comment:
I think all these variables were moved to private.  Can you remove them from this pull request?